lib/modules/datasource/custom/readme.md

```diff
@@ -83,10 +83,14 @@ All available options:
 
 ### Debugging
 
-Renovate writes tracing log entries before transformation starts and after if Renovate finds an unexpected data format.
-To surface this data you can use `logLevelRemap` to surface this data on the App or run Renovate in dryRun mode and log level set to `trace` locally.
+Renovate writes tracing logs entries before transformation.
+If Renovate finds an unexpected dataformat, it also writes a tracing log _after_ transformation.
 
-```json
+#### Getting trace level logs on hosted app
+
+If you use the Mend Renovate app, use the [`logLevelRemap` config option](https://docs.renovatebot.com/configuration-options/#loglevelremap) to get the trace log.
```

Comment:
Please:

- check if this is the correct config option
- suggest the correct link format